### PR TITLE
Improved metadata

### PR DESCRIPTION
**Pull request by demopaxlee**
As titled

### CODE REVIEW

File: core/src/components/Atom/MobileAppBar.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ function MobileAppBar(){
           <MenuIcon />
         </MenuButton>
         <AppBarTitle variant="h6">
-          FreedomHKG
+          ,Free Hong Kong
```
Comment (mingchuno) [inline core/src/components/Atom/MobileAppBar.js]:
```suggestion
          Free Hong Kong
```
???

**Comment by demopaxlee** [comment]
ping @mingchuno

**Comment by gaplo917** [comment]
@demopaxlee you installed the node dependencies in wrong location

should happen inside `core`

**Comment by gaplo917** [comment]
Ops, sorry. Not this PR

**Comment by demopaxlee** [comment]
so?

**Comment by demopaxlee** [comment]
Thanks

**Pull request merged**